client/src/ExamDetails.js

```diff
@@ -0,0 +1,13 @@
+function ExamDetails() {
+
+    return (
+        <>
+        <div>
+            <img src="https://medlineplus.gov/images/Xray_share.jpg"/>
```

Comment:
Is this for visual fe testing?

Reply:
Yes, so I can use it to adjust the card and image before calling the one exam.